pkgs/top-level/python-packages.nix

```diff
@@ -15579,16 +15579,25 @@ in modules // {
   };
 
 
-  pika = buildPythonPackage {
-    name = "pika-0.9.12";
-    disabled = isPy3k;
+  pika = buildPythonPackage rec {
+    name = "pika-${version}";
+    version = "0.10.0";
+
     src = pkgs.fetchurl {
-      url = https://pypi.python.org/packages/source/p/pika/pika-0.9.12.tar.gz;
-      sha256 = "670787ee6ade47cadd1ec8220876b9b7ae4df7bc4b9dd1d808261a6b47e9ce5d";
+      url = "https://pypi.python.org/packages/source/p/pika/${name}.tar.gz";
+      sha256 = "0nb4h08di432lv7dy2v9kpwgk0w92f24sqc2hw2s9vwr5b8v8xvj";
     };
-    buildInputs = with self; [ nose mock pyyaml ];
 
-    propagatedBuildInputs = with self; [ unittest2 ];
+    # Those use twisted which is only availalble for python2x
+    doCheck = !isPy3k;
```

Comment:
if that's the case, then this package won't be available for CPython 3.x anyway, so there is no need to add this.